src/CMakeLists.txt

```diff
@@ -314,6 +314,43 @@ if (WITH_BLKIN)
   add_subdirectory(blkin/blkin-lib)
 endif(WITH_BLKIN)
 
+## breakpad
```

Comment:
This definitely needs work to support non linux and possibly a build-time feature flag.